tests/system/provisioning/one_manager_agent/roles/manager-role/tasks/main.yml

```diff
@@ -79,7 +79,11 @@
 - name: Start Wazuh
   command: /var/ossec/bin/wazuh-control restart
 
-- name: Create script to check BD
+- name: Copy wdb-query.py script
   copy:
-    src: ../../../../../scripts/system_query_db.py
-    dest: /var/
+    src: ../../../../scripts/wdb-query.py
+    dest: /var/ossec/bin/wdb-query.py
+    owner: root
+    group: root
+    mode: '0644'
+    
```

Comment:
```suggestion

```

Reply:
The tab must be deleted in order to be detected correctly

Reply:
Added here https://github.com/wazuh/wazuh-qa/pull/5089/commits/d4744a39e1775ffe1bf805a0941ef8897af49805